library/Makefile

```diff
@@ -2,7 +2,7 @@ OPAM=opam
 EXEC=${OPAM} config exec
 DUNE=${EXEC} dune --
 
-RED_FILES=$(wildcard *.red)
+RED_FILES=$(wildcard ./**/*.red)
```

Comment:
@cangiuli @favonia @ecavallo

Reply:
(this just replicates the behavior of `master`)

Reply:
This is a great first step, but as we discussed, some way of defining an `all.red` file would avoid a lot of re-checking.

Reply:
@cangiuli Yes; since defining this `all.red` file in a non-stupid way requires some new functionality, I suggest that we do that on a different branch and get this merged shortly.